net/frr/src/opnsense/service/templates/OPNsense/Quagga/bgpd.conf

```
@@ -91,8 +91,9 @@ router bgp {{ OPNsense.quagga.bgp.asnumber }}
 {%   for neighbor in neighbors[addressFamily] %}
   neighbor {{ neighbor.address }} activate
 {%     if 'nexthopself' in neighbor and neighbor.nexthopself == '1' %}
-  neighbor {{ neighbor.address }} next-hop-self
+  neighbor {{ neighbor.address }} next-hop-self {% if 'nexthopselfall' in neighbor and neighbor.nexthopselfall == '1' %}all{%  endif %}
 {%     endif %}
+
```

```suggestion
  neighbor {{ neighbor.address }} next-hop-self {% if 'nexthopselfall' in neighbor and neighbor.nexthopselfall == '1' %}all{%  endif %}

{%     endif %}
```

shouldn't it be this way around?